src/server/ua_services_attribute.c

```diff
@@ -522,6 +522,17 @@ ReadWithNode(const UA_Node *node, UA_Server *server, UA_Session *session,
         retval = UA_Variant_setScalarCopy(&v->value, &node->variableNode.accessLevel,
                                           &UA_TYPES[UA_TYPES_BYTE]);
         break;
+    case UA_ATTRIBUTEID_ACCESSLEVELEX:
+        CHECK_NODECLASS(UA_NODECLASS_VARIABLE);
+        /* The normal AccessLevelEx contains the lowest 8 bits from the normal AccessLevel.
+         * In our case, all other bits are zero. */
+        UA_Byte value = (UA_Byte)(*(UA_UInt32*)v->value.data & 0xFF);
```

Comment:
This code looks fishy.
You want to wrap `node->variableNode.accessLevel` in a UInt32 and return that.
But you return a Byte. Wut?